### PR TITLE
Nodes reload secure settings

### PR DESCRIPTION
**Pull request by philkra**
as titled.

Need to complete the response (done).

### CODE REVIEW

File: specification/specs/cluster/reload_secure_settings/ReloadSecureSettingsRequest.ts [hunk 1]
```diff
@@ -25,5 +25,7 @@ class ReloadSecureSettingsRequest extends RequestBase {
   query_parameters?: {
     timeout?: Time
   }
-  body?: {}
+  body?: {
+    secure_settings_password: string
```
Comment (delvedor) [inline specification/specs/cluster/reload_secure_settings/ReloadSecureSettingsRequest.ts:29]: This should be optional :)
```suggestion
    secure_settings_password?: string
```

**Review by delvedor — CHANGES_REQUESTED** [review]

**Review by delvedor — COMMENTED** [review]
@philkra everything looks good here, can you regenerate the output?

**Comment by philkra** [comment]
@delvedor : https://github.com/elastic/elastic-client-generator/pull/108#pullrequestreview-593918504 --> done

**Review by delvedor — APPROVED** [review]
LGTM

**Pull request merged**